src/CosmosDB/CosmosDB/Restore/GetAzCosmosDBMongoDBRestorableDatabase.cs

```diff
@@ -0,0 +1,53 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.CosmosDB.Helpers;
+using Microsoft.Azure.Management.CosmosDB.Models;
+
+namespace Microsoft.Azure.Commands.CosmosDB
+{
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "CosmosDBMongoDBRestorableDatabase", DefaultParameterSetName = NameParameterSet), OutputType(typeof(PSRestorableMongodbDatabaseGetResult))]
+    public class GetAzCosmosDBMongoDBRestorableDatabase : AzureCosmosDBCmdletBase
+    {
+        [Parameter(Mandatory = true, ParameterSetName = NameParameterSet, HelpMessage = Constants.LocationNameHelpMessage)]
+        [ValidateNotNullOrEmpty]
+        public string LocationName { get; set; }
```

Comment:
LocationName -> Location

Reply:
We have two following two field which refers to different things as per current convention in code:
Location: array of string (like the set of locations where account need to be created)
LocationName: string (single location where we want to fetch that particular record.)

Help string is also per the above convention

Reply:
Why we aren't able to combine them as one?

Reply:
We can combine them as one but there are following two issue with that:
1. Get is only for one location. Allowing Get to take input string array might be bit confusing to customer and also we have to add checks to make sure and string array only contains 1 element in it.
2. Help message is different for Location:string[] vs LocationName:string.

Reply:
Updated name to Location